CONTRIBUTING.md

```diff
@@ -15,3 +15,18 @@ If you want to contribute to the Beats project, you can start by reading
 the [contributing guidelines](https://www.elastic.co/guide/en/beats/devguide/current/beats-contributing.html)
 in the _Beats Developer Guide_.
 
+## Dev Tools
```

Comment:
I'm concerned that we are splitting the content between this file and the Beats Developer Guide. Why not put this information into the "Contributing to Beats" topic in the dev guide instead? https://www.elastic.co/guide/en/beats/devguide/current/beats-contributing.html